test/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
 
-project(GreeterTests LANGUAGES CXX)
+project(
+  GreeterTests
+  VERSION 1.0
+  LANGUAGES CXX
+)
```

Comment:
I still don't think it makes sense to explicitly add the project version here and use `find_package` below.

- More code to maintain as the value is now present in two configuration files
- As said before the recommended way to test is on a clean system or root. That way only one version of the project is present no version conflict is possible.
- It wont protect anyone during development as the version is not updated with every code change.

I'd prefer if we could stick to the original implementation, which assumes that only one version is present.

Reply:
There should also a clear strategy to handle the case if find several version installed on host.
Dit you test it?

I have tested it and it does not work!

If this project is used as a template for a real production project, version upgrade, different version installed on  build host is a real use case!

Reply:
The `TEST_INSTALLED_VERSION` is only meant to test the install capabilities of the project and is usually only run in CI or isolated roots. For normal testing during development no installation is needed, so the problem shouldn't ever arise in the intended use-case.

For special requirements, there is nothing wrong with adjusting the starter for your projects or having a custom fork of it. However, to stay as general as possible, I prefer the starter remain minimal and not introduce code duplication for special use-cases.